core/node/storage/pg_stream_store.go

```diff
@@ -602,6 +642,7 @@ func (s *PostgresStreamStore) readMiniblocksTx(
 ) ([][]byte, error) {
 	miniblocksRow, err := tx.Query(
 		ctx,
+		// Do not escape this query as the specific partition may not exist
```

Comment:
For all "Do not escape" comments, does it make sense to handle "table not found" error in the same way as no results?

Reply:
I updated the handling for each specific case, ptal at the comments where I did not change things.